dspace-api/src/main/java/org/dspace/util/SolrImportExport.java

```diff
@@ -252,8 +252,9 @@ private static void reindex(String indexName, String exportDirName, boolean keep
         //The configuration details for the statistics shards reside within the "statistics" folder
         String instanceIndexName = indexName.startsWith("statistics-") ? "statistics" : indexName;
 
-        String solrInstanceDir = configurationService.getProperty("dspace.dir")
-                + File.separator + "solr" + File.separator + instanceIndexName;
+        String solrInstanceDir = configurationService.getProperty("solr.home") + File.separator + "server"
+                                + File.separator + "solr" + File.separator + "configsets"
+                                + File.separator + instanceIndexName + File.separator + "conf";
```

Comment:
Are you sure this should point at the "conf" subfolder?  It looks like it previously pointed at the instance directory (hence the name of the variable). I think the "conf" part can be removed here.